### PR TITLE
fixes names and math func.

### PR DESCRIPTION
**Pull request by bonj4**
changed bearing name to theta,
changed calculation fault of altitude #553 ,

### CODE REVIEW

File: calculation/intersection.py [hunk 1]
```diff
@@ -337,15 +337,16 @@ def intersection_points_find(self, **kwargs):
                     ph1 = 180 - ph1 if ph1 > 90 else ph1
                     ph2 = 180 - ph2 if ph2 > 90 else ph2
```
Comment (gorkemgul) [inline calculation/intersection.py:338]: Isn't that supposed to be deleted?

**Review by gorkemgul — COMMENTED** [review]
Aren't these lines supposed to be deleted?
```
ph1 = 180 - ph1 if ph1 > 90 else ph1
ph2 = 180 - ph2 if ph2 > 90 else ph2
```

File: calculation/intersection.py (hunk 1 above)
Comment (bonj4) [reply calculation/intersection.py]: thanks, changed

**Review by gorkemgul — COMMENTED** [review]
Problems seems to be fixed perfectly!

File: calculation/intersection.py (hunk 1 above)
Comment (gorkemgul) [reply calculation/intersection.py]: Mention not!

Comment (ozcan-durak) [reply calculation/intersection.py]: > Mention not!

what did you try to mean here :)

Comment (ozcan-durak) [reply calculation/intersection.py]: The main reason of the ph1 and ph2 to not look over 180° to 90° image.

**Pull request merged**